libraries/src/Document/Renderer/Feed/RssRenderer.php

```diff
@@ -236,6 +241,11 @@ public function render($name = '', $params = null, $content = null)
 				{
 					foreach ($data->items[$i]->category as $cat)
 					{
+						if ('ROOT' === $cat)
+						{
+							continue;
+						}
+						
```

Comment:
```suggestion
						}

```
Fix PHPCS (white space at the end of line).